easybuild/easyconfigs/u/umesimd/umesimd-0.8.1-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'CMakeMake'
+name = 'umesimd'
```

Comment:
We try to keep the naming based on how the project refers to itself.
:: in names would probably break things but UME-SIMD?